test/controllers/ReviewDetailsControllerSpec.scala

```diff
@@ -398,15 +305,7 @@ class ReviewDetailsControllerSpec extends PlaySpec with OneServerPerSuite with M
     test(result)
   }
 
-  private def continueWithAuthorisedAgent(service: String)(test: Future[Result] => Any) {
-    val userId = s"user-${UUID.randomUUID}"
-    builders.AuthBuilder.mockAuthorisedAgent(userId, mockAuthConnector)
-    FeatureSwitch.enable(FeatureSwitch("registration.usingGG", true))
-    when(mockBackLinkCache.saveBackLink(Matchers.any(), Matchers.any())(Matchers.any())).thenReturn(Future.successful(None))
-    when(mockAgentRegistrationService.enrolAgent(Matchers.any())(Matchers.any(), Matchers.any())).thenReturn(Future.successful(HttpResponse(OK)))
-    val result = testReviewDetailsController(nonDirectMatchReviewDetails).continue(service).apply(fakeRequestWithSession(userId))
-    test(result)
-  }
+
 
   private def continueWithAuthorisedAgentEMAC(service: String)(test: Future[Result] => Any) {
```

Comment:
Should this test still make reference to a FeatureSwitch registration.usingGG if we are removing it?